src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -393,6 +395,34 @@ public ClusterState sendClusterStateRequest(TransportService transportService) {
         return clusterStateResponseHandler.getClusterState();
     }
 
+    /**
+     * Request the Dependency Information from Opensearch.
+     * @param transportService
+     * @return The dependency information of Extension
+     */
+
+    public List<DiscoveryExtensionNode> sendExtensionDependencyRequest(TransportService transportService) {
+        logger.info("Sending Extension Dependency Information request to Opensearch");
+        ExtensionDependencyResponseHandler extensionDependencyResponseHandler = new ExtensionDependencyResponseHandler();
+        try {
+            transportService.sendRequest(
+                opensearchNode,
+                ExtensionsManager.REQUEST_EXTENSION_DEPENDENCY_INFORMATION,
+                new ExtensionRequest(ExtensionsManager.RequestType.REQUEST_EXTENSION_DEPENDENCY_INFORMATION),
```

Comment:
You should be using the constructor that passes the unique ID here!

Reply:
This needs the `uniqueId` as a second parameter to the `ExtensionRequest` constructor.